### DISCUSSION TITLE
MUI Autocomplete cascade

### DISCUSSION THREAD

**Discussion by vicasas** [react-hook-form/react-hook-form#4047] [Q&A]
I am trying to make a cascade form with the Autocmplete component but I have some warning in the console when changing a value that from component 1 since the second value that is selected is still the previous one, I cannot clean this, I am using setValue and it works, but the warning still appears from MUI. Is there an example of how to do this?

**Comment by bluebill1049** [comment]
I am taking a look at your issue. (in the issues tab)

**Reply by bluebill1049** [reply]
I just reviewed your issue, isn't it the warning coming from MUI?

**Reply by vicasas** [reply]
Exactly, it is the warning that Material UI throws. Maybe we should close this and follow the issue thread. #4032
